spree_email_to_friend.gemspec

```diff
@@ -35,10 +35,12 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'email_spec', '>= 1.6.0'
   s.add_development_dependency 'simplecov', '~> 0.8.2'
   s.add_development_dependency 'database_cleaner', '~> 1.4.0'
-  s.add_development_dependency 'coffee-rails', '~> 4.0.0'
+  s.add_development_dependency 'coffee-rails', '~> 4.2.1'
   s.add_development_dependency 'sass-rails', '~> 5.0.0'
   s.add_development_dependency 'ffaker', '>= 1.32.0'
   s.add_development_dependency 'guard-rspec', '>= 4.3.1'
   s.add_development_dependency 'pry-rails', '>= 0.3.2'
   s.add_development_dependency 'rubocop', '>= 0.24.1'
+  s.add_development_dependency 'rabl', '= 0.13.0'
```

Comment:
It’s because of monkey path at https://github.com/spree/spree/issues/7649
It was fixed today. Without it rake test_app would not work. Rabl removed.